src/hooks/useErc20.ts

```diff
@@ -34,7 +35,8 @@ async function _fetchErc20FromNetwork(params: {
   }
 
   try {
-    return getErc20Info({ tokenAddress: address, networkId, web3, erc20Api })
+    if (Math.floor(Math.random() * 10) < 5) return null
```

Comment:
I have applied randomization so that some cases return null and force the provider update.